marketplace/gatsby-preview/src/AppConfig.js

```diff
@@ -47,17 +47,20 @@ export default class AppConfig extends React.Component {
     const params = installationParams || {};
 
     // eslint-disable-next-line react/no-did-mount-set-state
-    this.setState(prevState => {
-      return {
-        checkedContentTypes: items.reduce((acc, ct) => {
-          acc[ct.sys.id] = { name: ct.name, checked: previouslyCheckedTypes.includes(ct.sys.id) };
-          return acc;
-        }, prevState.checkedContentTypes),
-        previewUrl: params.previewUrl || '',
-        webhookUrl: params.webhookUrl || '',
-        authToken: params.authToken || ''
-      };
-    });
+    this.setState(
+      prevState => {
+        return {
+          checkedContentTypes: items.reduce((acc, ct) => {
```

Comment:
Is `acc` an array or an object? Something is off here.

Reply:
`acc` is an object in this sense which looks something like `{name: string, checked: boolean}` and is built off the previous state, `prevState.checkedContentTypes`. I actually didn't change any of this logic here, I only added `() => app.setReady()`. Prettier took over the rest...

Reply:
Ah true, I misunderstood the code.

I'd still do:

```
return {
  ...acc,
  [ct.sys.id]: { name: ct.name, checked: previouslyCheckedTypes.includes(ct.sys.id)
}
```

so we don't mutate `acc`

Reply:
Changed here: https://github.com/contentful/extensions/pull/267/commits/b36fb56f8ac178c078978726487fb8d2be86b3b7